nixos/modules/system/boot/unl0kr.nix

```diff
@@ -10,10 +16,32 @@ in
         Whether to enable the unl0kr on-screen keyboard in initrd to unlock LUKS.
       '';
     };
+
+    settings = lib.mkOption {
+      description = ''
+        Configuration for `unl0kr`.
+
+        See `unl0kr.conf(5)` for supported values.
+
+        Alternatively, visit `https://gitlab.com/postmarketOS/buffybox/-/blob/unl0kr-2.0.0/unl0kr.conf`
+      '';
+
+      example = lib.literalExpression ''
+        {
+            general.animations = true;
+            theme = {
+            default = "adwaita-dark";
```

Comment:
Nitpick, but the indentation seems to be off on this line.

Reply:
that's odd this is outdated

Reply:
resolved, with the appropriate settings for that version